lib/asset_sync/config.rb

```diff
@@ -37,7 +37,7 @@ class Invalid < StandardError; end
     attr_reader   :fog_public            # e.g. true, false, "default"
 
     # Amazon AWS
-    attr_accessor :aws_access_key_id, :aws_secret_access_key, :aws_session_token, :aws_reduced_redundancy, :aws_iam_roles, :aws_signature_version
+    attr_accessor :aws_access_key_id, :aws_secret_access_key, :aws_session_token, :aws_reduced_redundancy, :aws_iam_roles, :aws_signature_version, :aws_acl
```

Comment:
I think it's time to make this one line per attribute 
As it becomes too long to read if not
```suggestion
    attr_accessor :aws_access_key_id
    attr_accessor :aws_secret_access_key
    attr_accessor :aws_session_token
    attr_accessor :aws_reduced_redundancy
    attr_accessor :aws_iam_roles
    attr_accessor :aws_signature_version
    attr_accessor :aws_acl

    # Fog
```
The blank line is intended